doc/pages/user/060-using_triggers.md

```diff
@@ -245,3 +246,14 @@ If the Trigger is installed, when a device sends data to the interface/path defi
   }
 }
 ```
+
+## Restricting triggers to a single device or group
+
+Both device and data triggers accept the `device_id` and `group_name` keys to restrict a trigger to
+a single device or a single group.
+
+Triggers containing the `device_id` key will be triggered only for the specified device, while
+triggers containing the `group_name` key will be triggered only if the device is member of the group
+that is indicated in the `group_name` key. Note that when changes are made to the groups (_e.g._ a
```

Comment:
Note that when devices in a group are added or removed, the changes are not reflected immediately in group
triggers [...]